src/components/DiagramFrame/SeqDiagram/SeqDiagram.vue

```diff
@@ -7,20 +7,36 @@
          .bg-skin-base is repeated because .zenuml reset it to default theme.
      -->
     <div :style="{ paddingLeft: `${paddingLeft}px` }" class="relative">
-      <life-line-layer
-        :leftGap="paddingLeft"
-        :context="rootContext.head()"
-        :renderParticipants="false"
-      />
-      <message-layer
-        :context="rootContext.block()"
-        :style="{ width: `${width}px` }"
-      />
-      <life-line-layer
-        :leftGap="paddingLeft"
-        :context="rootContext.head()"
-        :renderParticipants="true"
-      />
+      <slot v-if="mode === 'dynamic'">
```

Comment:
Why we need a slot here, I think it should be a `div`

Reply:
Good catch. I misused the slot tag. I replaced it with the template tag instead.